stl/inc/functional

```diff
@@ -1536,12 +1523,11 @@ public:
         : _Mypair(_One_then_variadic_args_t{}, _STD forward<_Fx>(_Func), _STD forward<_Types>(_Args)...) {}
 
     template <class... _Unbound>
-        constexpr auto operator()(_Unbound&&... _Unbargs) //
+    constexpr auto operator()(_Unbound&&... _Unbargs) //
         & noexcept(noexcept( //
             _Call_front_binder(_Seq{}, _Mypair._Get_first(), _Mypair._Myval2, _STD forward<_Unbound>(_Unbargs)...))) //
-          -> decltype( //
-              _Call_front_binder(
-                  _Seq{}, _Mypair._Get_first(), _Mypair._Myval2, _STD forward<_Unbound>(_Unbargs)...)) { //
+        -> decltype( //
+            _Call_front_binder(_Seq{}, _Mypair._Get_first(), _Mypair._Myval2, _STD forward<_Unbound>(_Unbargs)...)) { //

```

Comment:
Empty comments were being used here to improve clang-format's behavior. If you remove them, does clang-format 10 do something reasonable-looking now?

Reply:
I admittedly didn't even read what clang-format did :P

Reply:
That's what I'm for, to read the output of humans and robots alike, line by repetitive line, until I find the single wrong one.